src/v2/guide/components.md

```diff
@@ -507,7 +507,7 @@ We have learned that the parent can pass data down to the child using props, but
 Every Vue instance implements an [events interface](../api/#Instance-Methods-Events), which means it can:
 
 - Listen to an event using `$on(eventName)`
-- Trigger an event using `$emit(eventName)`
+- Trigger an event using `$emit(eventName[, optionalPayload])`
```

Comment:
This should be `$emit(eventName, [optionalPayload])`

Reply:
Though, truthfully, it doesn't need to be in an array, so maybe that should be more clear, too.

Reply:
@sdras I put it into square brackets to specify it is optional, do you have any suggestion on the format?
If you think it's better, I can write it in this way
>  Trigger an event using `$emit(eventName)` or `$emit(eventName, optionalPayload)`
In order to make it explicit ;)

Reply:
Or I can edit as
> Trigger an event using `$emit(eventName, optionalPayload)`

if it's clear enough